### PR TITLE
Added Jenkinsfile

### PR DESCRIPTION
**Pull request by todorinskiz**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    sh "./gradlew publish -Pde_publish_username=${SONNATYPE_CREDENTIALS_USR} -Pde_publish_password=${SONNATYPE_CREDENTIALS_PSW} -PkeyId=DF8285F0 -Ppassword=${GPGPASSPHRASE} -PsecretKeyRingFile=/var/jenkins_home/secring.gpg"
+                }
+            }
+        }
+    }
+
+
+    post {
+        success {
+            archiveArtifacts artifacts: '**/build/dist/*.amp'
```
Comment (kerkhofsd) [inline Jenkinsfile:40]: Do we still do this?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        stage("Clean") {
+            steps {
+                sh "./gradlew clean"
+            }
+        }
+
+        stage("Assemble") {
+            steps {
+                sh "./gradlew amp"
```
Comment (kerkhofsd) [inline Jenkinsfile:14]: `Assemble` != `amp`?

I'd expect that a `./gradlew assemble` builds the amp artifact as well?

File: Jenkinsfile (hunk 1 above)
Comment (tgeens) [reply Jenkinsfile:40]: No

File: Jenkinsfile (hunk 2 above)
Comment (driesdeproost) [reply Jenkinsfile:14]: yes

**Comment by todorinskiz** [comment]
Closing in favor of Travis CI.
Will include the feedback in the next PR

**Pull request closed without merging**